xcode/X16/iOS/ios_functions.c

```diff
@@ -0,0 +1,51 @@
+//
```

Comment:
yay, msgSend magic! :)

But I don't see why this can't be ios_functions.m and use ObjC? The interface to the main code can still be the same, using a C interface.

Reply:
At the moment the observer of notifications sent between .c and SDL needs to be created through msgsend. I removed the notification msgsend though.